### PR TITLE
lesson 11: feat: added modal for main-section to choose accommodation…

### PR DESCRIPTION
**Pull request by LudmilaZhuleha**
… conditions, added listeners and counter functionality.

Please have a look at closeModal() and listener, which uses it. It does not work. I can not find out why.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,31 +1,111 @@
...
+  counterPlus(countBtnRooms, 30);
+});
+
+downBtnRooms.addEventListener('click', (e) => {
+  e.preventDefault();
+  counterMinus(countBtnRooms, 30);
+});
+upBtnChilds.addEventListener('click', (e) => {
+  e.preventDefault();
+  counterPlus(countBtnChilds, 10);
```
Comment (Marinarium) [inline src/main.js:92]: You have to have 10 limit for children, hovewer + button still work and add options. You can add disabled for this button for example
Screenshot 2022-07-08 at 12 00 34

This task is not about css, but try to overlap the second section title

File: src/main.js [hunk 2]
```diff
@@ -1,31 +1,111 @@
...
+function openModal() {
+  modalConditions.style.display = 'block';
+}
+function closeModal() {
+  modalConditions.style.display = 'none';
+}
+inputConditions.addEventListener('click', openModal);
+
+window.addEventListener('click', (event) => {
+  if (!event.target.contains('.main-form-modal')) {
```
Comment (Marinarium) [inline src/main.js]: Please watch console when you are testing your code. This error clearly describe why you can't close a modal
Screenshot 2022-07-08 at 12 19 46
You should use node for contains function argument
```
  if (event.target.contains(modalConditions)) {
    closeModal();
  }
```

File: src/main.js [hunk 3]
```diff
@@ -1,31 +1,111 @@
...
+upBtnRooms.addEventListener('click', (e) => {
+  e.preventDefault();
+  counterPlus(countBtnRooms, 30);
+});
+
+downBtnRooms.addEventListener('click', (e) => {
+  e.preventDefault();
+  counterMinus(countBtnRooms, 30);
+});
+upBtnChilds.addEventListener('click', (e) => {
```
Comment (Marinarium) [inline src/main.js:90]: please add empty space to prettify)

File: src/main.js (hunk 1 above)
Comment (LudmilaZhuleha) [reply src/main.js:92]: Added if statement limiting select further spread after 10 clicks.

File: src/main.js (hunk 2 above)
Comment (LudmilaZhuleha) [reply src/main.js]: Done.

File: src/main.js (hunk 3 above)
Comment (LudmilaZhuleha) [reply src/main.js:90]: Done.

**Pull request merged**